packages/analytics/src/helpers.ts

```diff
@@ -206,3 +203,16 @@ export function wrapOrCreateGtag(
     wrappedGtag: window[gtagFunctionName] as Gtag
   };
 }
+
+/**
+ * Returns first script tag in DOM matching our gtag url pattern.
+ */
+export function findGtagScriptOnPage(): HTMLScriptElement | null {
+  const scriptTags = window.document.getElementsByTagName('script');
+  for (const tag of Object.values(scriptTags)) {
```

Comment:
You an iterate directly on `scriptTags` - `for (const tag of scriptTags)`

Reply:
Confirmed we cannot do this without `downlevelIteration` and adding `dom.iterable` TS lib, which we agreed adds unnecessary bloat to the analytics package.